components/support/error/tests/macro_arguments.rs

```diff
@@ -34,10 +34,35 @@
      }
  }
 
- #[handle_error(String)]
- fn func() -> ::std::result::Result<String, ExternalError> {
-     Err(Error{})
- }
+#[handle_error(Error)] // Works.
```

Comment:
nit: Can we add another failing case of `#[handle_error(String)]` or some other type that isn't the correct one?